DataFormats/ForwardDetId/interface/HGCScintillatorDetId.h

```diff
@@ -0,0 +1,70 @@
+#ifndef DataFormats_ForwardDetId_HGCScintillatorDetId_H
+#define DataFormats_ForwardDetId_HGCScintillatorDetId_H 1
+
+#include <iosfwd>
+#include "DataFormats/DetId/interface/DetId.h"
+#include "DataFormats/ForwardDetId/interface/ForwardSubdetector.h"
+
+class HGCScintillatorDetId : public DetId {
+
+public:
+
+  /** Create a null cellid*/
+  HGCScintillatorDetId();
+  /** Create cellid from raw id (0=invalid tower id) */
+  HGCScintillatorDetId(uint32_t rawid);
+  /** Constructor from subdetector, zplus, layer, module, cell numbers */
+  HGCScintillatorDetId(int type, int layer, int ieta, int iphi);
+  /** Constructor from a generic cell id */
+  HGCScintillatorDetId(const DetId& id);
+  /** Assignment from a generic cell id */
+  HGCScintillatorDetId& operator=(const DetId& id);
+  
+  /** Converter for a geometry cell id */
+  HGCScintillatorDetId geometryCell () const {return HGCScintillatorDetId (type(), layer(), ieta(), 0);}
```

Comment:
How is this function used? (It seems the old HGC DetIds have it too, but I'm not sure exactly what it does.)

Reply:
This is used in geometry function - because in geometry we define a larger object than a small cell